### PR TITLE
Slovenian, Ukrainian localizations

### PR DESCRIPTION
**Pull request by 1ec5**
Added partial Slovenian and Ukrainian localizations from Transifex. Neither language is supported by Polly.

/cc @bsudekum @Andygol

### CODE REVIEW

File: MapboxNavigation.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -319,7 +319,6 @@
 		351BEBE81E5BCC63006FE110 /* RouteTableViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = RouteTableViewController.swift; sourceTree = "<group>"; };
 		351BEBE91E5BCC63006FE110 /* RouteTableViewHeaderView.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = RouteTableViewHeaderView.swift; sourceTree = "<group>"; };
 		351BEBEA1E5BCC63006FE110 /* NavigationViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; lineEnding = 0; path = NavigationViewController.swift; sourceTree = "<group>"; xcLanguageSpecificationIdentifier = xcode.lang.swift; };
-		351BEBEB1E5BCC63006FE110 /* String.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = String.swift; sourceTree = "<group>"; };
```
Comment (bsudekum) [inline MapboxNavigation.xcodeproj/project.pbxproj]: This strings file removal seems to be happening on all PRs...

Comment (1ec5) [reply MapboxNavigation.xcodeproj/project.pbxproj]: Yeah, I think it’s Xcode belatedly cleaning up orphaned references to the file or something.

**Comment by 1ec5** [comment]
9bfe2e59b1acc1045f9c7bf535174e7f3c1466b4 upgrades to OSRM Text Instructions v0.3.2.

**Comment by bsudekum** [comment]
Looks good.

**Comment by 1ec5** [comment]
2da4bbeae61b550aef315a078859bd38225daaaa sets the voice for Italian, now that OSRM Text Instructions includes an Italian localization.

**Pull request merged**